girder/web_client/src/views/widgets/ItemListWidget.js

```diff
@@ -140,6 +147,106 @@ var ItemListWidget = View.extend({
+
+            this.observer = new MutationObserver((mutations) => {
+                // for every mutation
+                _.each(mutations, (mutation) => {
+                    // If no nodes are added we can return
+                    if (!mutation.addedNodes) {
+                        return;
+                    }
+
+                    // for every added node
+                    _.each(mutation.addedNodes, (node) => {
+                        if (node.className && node.className.indexOf('g-item-list') !== -1) {
+                            // We want to do a onetime scroll to position if the screen is idle
+                            if (window.requestIdleCallback) {
+                                // Processing time is 2 seconds, but should finish much faster
+                                requestIdleCallback(() => {
+                                    this.centerSelected();
+                                }, { timeout: 2000 });
+                            } else {
+                                this.centerSelected();
+                            }
+                        }
+
+                        // For any images added we wait until loaded and rescroll to the selected
+                        if (_.isFunction(node.getElementsByTagName)) {
+                            $('img', node).on('load', onLoadImage);
+                        }
+                    });
+                });
+            });
+
+            // bind mutation observer to a specific element (probably a div somewhere)
+            this.observer.observe(target.parent()[0], { childList: true, subtree: true });
+
+            // Remove event listeners and disconnect observer
+            const unbindDisconnect = () => {
+                if (this.observer) {
+                    widgetcontainer.unbind('scroll.observerscroll');
+                    widgetcontainer.unbind('wheel.observerscroll');
+                    this.observer.disconnect();
+                    this.observer = null;
+                    this.tempScrollPos = undefined;
+                }
+            };
+
+            // Add in scroll event to monitor the scrollPos to prevent uncessary updates and also kill observer when user scrolls
+            widgetcontainer.bind('scroll.observerscroll', (evt) => {
+                if (this.tempScrollPos !== undefined && this.tempScrollPos !== widgetcontainer[0].scrollTop) {
+                    this.tempScrollPos = widgetcontainer[0].scrollTop;
+                    // If the event detail is not 'selected_item_scroll' the scroll is should be a user initiated scroll
+                    if (evt.detail !== 'selected_item_scroll') {
+                        unbindDisconnect();
+                    }
+                }
+            });
+            // Backup function to kill observer if user moves scrollwheel, or touchpad equivalent
+            widgetcontainer.bind('wheel.selectionobserver', (evt) => {
+                unbindDisconnect();
+            });
```

Comment:
This can now be `widgetcontainer.bind('wheel.selectionobserver', unbindDisconnect);`